doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -121,15 +125,19 @@ To read the logs from the :ref:`multicore_hello_world` sample programmed to the
 
      .. code-block:: console
 
-        *** Booting Zephyr OS build v2.7.99-ncs1-2193-gd359a86abf14  ***
+        *** Booting nRF Connect SDK zephyr-v3.5.0-3517-g9458a1aaf744 ***
         Hello world from nrf54h20dk_nrf54h20_cpuapp
+        Hello world from nrf54h20dk_nrf54h20_cpuapp
```

Comment:
the logs changed from dk to pdk too. in which case this will need some update in tests anyway